### DISCUSSION TITLE
build-failed vcpkg install boost

### DISCUSSION THREAD

**Discussion by zhoujd00** [microsoft/vcpkg#26081] [Q&A]
******************host env:
 win7-x64 
 vs2017enterprise win10sdk
**************operation:
vcpkg install --triplet=x64-windows websocketpp
***************question**************************
cl not found?
PATH has already add cl.exe's path，i can execute cl rigthly in cmd. why vcpkg install failed?->bootstrap.bat msvc -->config_toolset.bat 
************************console error
-- Bootstrapping...
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:131 (message):

    Command failed: D:/GitHub/vcpkg/packages/boost-build_x64-windows/tools/boost
-build/bootstrap.bat msvc
    Working Directory: D:/GitHub/vcpkg/packages/boost-build_x64-windows/tools/bo
ost-build
    Error code: 1
    See logs for more information:
      D:\GitHub\vcpkg\buildtrees\boost-build\bootstrap-x64-windows-out.log
      D:\GitHub\vcpkg\buildtrees\boost-build\bootstrap-x64-windows-err.log

Call Stack (most recent call first):
  ports/boost-build/portfile.cmake:66 (vcpkg_execute_required_process)
  scripts/ports.cmake:147 (include)
*************************************bootstrap-x64-windows-out.log
Building the B2 engine..
Call_If_Exists "bin\VCVARS32.BAT"
###
###

D:\GitHub\vcpkg\packages\boost-build_x64-windows\tools\boost-build\src\engine>"cl" /nologo /MP /MT /TP /Feb2 /wd4996 /O2 /GL /EHsc   -DNDEBUG  builtins.cpp class.cpp command.cpp compile.cpp constants.cpp cwd.cpp debug.cpp debugger.cpp execcmd.cpp execnt.cpp execunix.cpp filent.cpp filesys.cpp fileunix.cpp frames.cpp function.cpp glob.cpp hash.cpp hcache.cpp hdrmacro.cpp headers.cpp jam.cpp jamgram.cpp lists.cpp make.cpp make1.cpp md5.cpp mem.cpp modules.cpp native.cpp object.cpp option.cpp output.cpp parse.cpp pathnt.cpp pathsys.cpp pathunix.cpp regexp.cpp rules.cpp scan.cpp search.cpp jam_strings.cpp startup.cpp subst.cpp sysinfo.cpp timestamp.cpp variable.cpp w32_getreg.cpp modules/order.cpp modules/path.cpp modules/property-set.cpp modules/regex.cpp modules/sequence.cpp modules/set.cpp /link kernel32.lib advapi32.lib user32.lib 

D:\GitHub\vcpkg\packages\boost-build_x64-windows\tools\boost-build\src\engine>dir *.exe 
 Volume in drive D is 软件
 Volume Serial Number is 0009-616A

 Directory of D:\GitHub\vcpkg\packages\boost-build_x64-windows\tools\boost-build\src\engine

D:\GitHub\vcpkg\packages\boost-build_x64-windows\tools\boost-build\src\engine>copy /b .\b2.exe .\bjam.exe 
The system cannot find the file specified.

Failed to build the B2 engine.
**************************************************bootstrap-x64-windows-err.log
'"cl"' is not recognized as an internal or external command,
operable program or batch file.
File Not Found

**Comment by JackBoosY** [comment]
Fill an issue instead of a discussion.
